cmd-info.c

```diff
@@ -785,6 +785,11 @@ int command_info(int argc, char *argv[], struct opts *opts)
 			.flags = SYMTAB_FL_DEMANGLE,
 		};
 
+		if (!opts->exename) {
+			pr_out("Usage: uftrace info [--symbols] [COMMAND]\n");
```

Comment:
I modified this PR prechecking symbols option before the "no data" warning.
But if you don't want to add this usage, I'll remove it.